### DISCUSSION TITLE
How to configure with Webpack?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#2103] [Q&A] [answered]
Hello,
I'm having a hard time configuring Prisma to run with Webpack.

I know that the query engine binary file has to be copied along with the bundled code - so that it can access it easily.

Does anyone have an example of this working with Webpack?

I've tried using webpack-copy-plugin to copy it, and it seems that it gets found correctly (no warning about not being able to find the specified file, like before) - but then I receive an ENOENT error when it tries to spawn the binary.

I imagine that something else has to be copied aswell, or some setting has to be changed so that it can find everything it needs.

Could someone set-up an extremely basic example of Prisma and Webpack working together?

**Comment by nikolasburk** [comment, marked as answer]
Hey @TheDumpap, thanks a lot for raising this! Have you seen this example already: https://github.com/prisma/prisma2-e2e-tests/tree/master/bundlers/webpack Please let me know if that addresses your issue (and if it does, please mark this comment as an answer to the question) 🙂
